### DISCUSSION TITLE
How to verify a string TYPE only and ignore its value

### DISCUSSION THREAD

**Discussion by nabeel-servcorp** [VerifyTests/Verify#910] [Q&A]
I know that the library provides some helpful methods for example `ScrubMember()`, `ScrubMembersWithType()`, `IgnoreMember()` etc. but neither of these is working for me because I am trying to verify an API response which is an auto-generated plain string. It is NOT a JSON response so none of the above-mentioned methods work. For now I tackled it with using `ScrubLinesContaining("SM-")` but I am wondering if there is any other solution in which I ignore the value of the string and only verify if the response is of type `string.

Because this string is not a GUID so I can't use `ScrubInlineGuids()` method.

**Comment by SimonCropp** [comment]
can u share the code of your test. so i can better understand the scenario

**Reply by nabeel-servcorp** [reply]
> can u share the code of your test. so i can better understand the scenario

Here is my xUnit test method for the scenario. I have omitted irrelevant details for brevity.
```c#
[Fact]
public async Task CreateResource_Succeeds()
{
        var apiResponse = await _apiClient.CreateResource(requestModel); //apiResponse will be a plain non-JSON and non-GUID string
        Assert.NotNull(apiResponse);
        await Verifier
            .Verify(apiResponse)
            .ScrubLinesContaining("SM-");
}
```

**Reply by SimonCropp** [reply]
and what will the string contain?

**Reply by nabeel-servcorp** [reply]
Some auto-generated strings like "_SM-abc123_", "_SM-xyz456_", "_SM-blablabla_" etc.

**Reply by SimonCropp** [reply]
can you submit a failing unit test as a pull request
